pkg/common/configs/configs.go

```diff
@@ -26,11 +26,34 @@ import (
 )
 
 const (
-	HealthCheckInterval = "health.checkInterval"
+	// prefixes
+	PrefixEvent  = "event."
+	PrefixHealth = "health."
+
+	HealthCheckInterval = PrefixHealth + "checkInterval"
+
+	// events
+	CMEventRequestsEnabled = PrefixEvent + "requestsEnabled"
+	CMEventTrackingEnabled = PrefixEvent + "trackingEnabled"
+	CMEventRequestCapacity = PrefixEvent + "requestCapacity"
+	CMEventBufferCapacity  = PrefixEvent + "bufferCapacity"
+	CMEventResponseSize    = PrefixEvent + "responseMaxSize"
+
+	// defaults
+	DefaultHealthCheckInterval     = 30 * time.Second
+	DefaultEventRequestsEnabled    = false
+	DefaultEventTrackingEnabled    = true
+	DefaultEventRequestCapacity    = 1000
+	DefaultEventBufferCapacity     = 100000
+	DefaultEventResponseSize       = 10000
```

Comment:
This is not easy to enforce. We need to serialise the data  before we know the final size and cannot add or remove anything from that serialised buffer. Think we need to drop this and just use the request capacity.